index.html

```diff
@@ -147,151 +141,18 @@
     contains.
   </p>
   <p>
-    Implementations that use [[!ECMAScript]] to implement the APIs defined in
+    Implementations that use [[ECMAScript]] to implement the APIs defined in
```

Comment:
Separate PR... but delete this whole statement. It serves no purpose and I don't know why people keep including it.

Reply:
(I know also you didn't add it :) ... just constantly having to delete it from specs!)

Reply:
For sure!